setup.py

```diff
@@ -96,7 +96,10 @@ def read(fname):
     setup(
         name="err",
         version=VERSION,
-        packages=find_packages(src_root),
+        packages=find_packages(src_root, exclude=['tests',
+                                                  'tests.*',
```

Comment:
@gbin currently we do not, but I would leave them there, just to be safe.

If you do not agree I am happy to remove them.